javascript2/week1/script.js

```diff
@@ -0,0 +1,123 @@
+let recipeObject = {
+  id: 1,
+  title: "Vegetable Biryani",
+  picture_url: "./images/veg_biryani.jpeg",
```

Comment:
Minor: Looks like we're not using the .webp version of the image. It's not a big deal, but in the name of keeping our workspace tidy, I would recommend removing unused files!